headers/components/component_names.hpp

```diff
@@ -26,6 +26,7 @@
 #define OH_COMPONENT_NAME   "OH"
 #define N2O_COMPONENT_NAME "N2O"
 #define TEMPERATURE_COMPONENT_NAME "temperature"
```

Comment:
If we are removing the old temperature component, then shouldn't we also remove this `#define`?

Reply:
done